src/main/features/response/views/eligibility/how-much-paid-claimant.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set title = t('Money claim') %}
+
+{% set heading = t('How much have you paid the claimant?') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" class="analytics-submit-event-trigger" data-event-action="Response: How much have you paid the claimant option" data-event-label-from="option" novalidate>
```

Comment:
break this into multiple lines so that it can easily be read at a glance without scrolling
the action shouldn't include the text 'option' as that isn't the action taken, please remove the word, consider adding a question mark to the end
REQ